### ISSUE TITLE
its not good with new schema version

### ISSUE THREAD

**Issue by mosisms** [kamalkhan/kk-star-ratings#88]
## Detailed description

recently new schema roles are produced with google and after that we see some error in schema code of this plugin in google test.
please check and update that

**Comment by QasimYad** [comment]
> Since Google’s September 2019 update, you may see this error when trying to validate your review schema; “Thing is not a known valid target type for the itemReviewed property”

**Comment by mosisms** [comment]
Exactly this error was shown.
How we can resolve it?

**Comment by DanBusc** [comment]
I'm getting a warning from Google because the title contains quotation marks 

>"@type": "CreativeWorkSeries", 
"name" : "this is "a" test",
....
Please escape or remove quotation marks.

many thanks!

**Comment by bilalnaseer** [comment]
Schema is not showing at all, I have tested on several pages. Some one else having the same issue?

**Comment by bilalnaseer** [comment]
> Schema is not showing at all, I have tested on several pages. Some one else having the same issue?

This is issue is resolved once I downloaded version 3.1.1 from Github repo but when I update it from WordPress repository 4.1.2 it does not work any more

**Comment by bilalnaseer** [comment]
Okay so the latest version 4.1.2 in WordPress repository has bug here http://prntscr.com/pwqrp0 (blank Json format) which should be below:

```
{
    "@context": "https://schema.org/",
    "@type": "CreativeWorkSeries",
    "name": "[title]",
    "aggregateRating": {
        "@type": "AggregateRating",
        "ratingValue": "[score]",
        "bestRating": "[best]",
        "ratingCount": "[count]"
    }
}
```

I hope others will benefit from my comment @kamalkhan quick fix I guess
